config_defaults.yml

```diff
@@ -1,29 +1,72 @@
 ---
 .travis.yml:
-  script: "\"bundle exec rake validate lint spec\""
+  supported_oss: &supported_oss
+    puppetlabs-acl:                      [windows]
+    puppetlabs-chocolatey:               [windows]
+    puppetlabs-dism:                     [windows]
+    puppetlabs-dsc:                      [windows]
+    puppetlabs-iis:                      [windows]
+    puppetlabs-powershell:               [windows]
+    puppetlabs-reboot:                   [windows]
+    puppetlabs-registry:                 [windows]
+    puppetlabs-sqlserver:                [windows]
+    puppetlabs-wsus_client:              [windows]
+    puppetlabs-inifile:                  [linux,windows]
+    puppetlabs-stdlib:                   [linux,windows]
+    puppetlabs-accounts:                 [linux]
+    puppetlabs-apache:                   [linux]
+    puppetlabs-apt:                      [linux]
+    puppetlabs-concat:                   [linux]
+    puppetlabs-firewall:                 [linux]
+    puppetlabs-haproxy:                  [linux]
+    puppetlabs-hocon:                    [linux]
+    puppetlabs-ibm_installation_manager: [linux]
+    puppetlabs-inifile:                  [linux]
+    puppetlabs-java:                     [linux]
+    puppetlabs-java_ks:                  [linux]
+    puppetlabs-mongodb:                  [linux]
+    puppetlabs-motd:                     [linux]
+    puppetlabs-mysql:                    [linux]
+    puppetlabs-ntp:                      [linux]
+    puppetlabs-postgresql:               [linux]
+    puppetlabs-rabbitmq:                 [linux]
+    puppetlabs-stdlib:                   [linux]
+    puppetlabs-tagmail:                  [linux]
+    puppetlabs-tftp:                     [linux]
+    puppetlabs-tomcat:                   [linux]
+    puppetlabs-vcsrepo:                  [linux]
+    puppetlabs-vsphere:                  [linux]
+    puppetlabs-xinetd:                   [linux]
+  script: "\"bundle exec rake release_checks\""
+  docker_sets:
+  - set: docker/ubuntu-14.04
+  - set: docker/centos-7
+  docker_defaults:
+    # values will replace @@SET@@ with the docker_sets' value
+    rvm: 2.3.1
+    sudo: required
+    dist: trusty
+    services: docker
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_set=@@SET@@
+    script: bundle exec rake beaker
   includes:
-  - rvm: 1.9.3
-    env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 2.0.0
-    env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 2.1.5
-    env: PUPPET_GEM_VERSION="~> 4.0"
   - rvm: 2.3.1
     env: PUPPET_GEM_VERSION="~> 4.0"
+    bundler_args: --without system_tests
+  - rvm: 2.1.5
```

Comment:
@ferventcoder What do you say about this? I took the ruby versions from the windows branch as they are more specifically chosen I believe.